wfe2/wfe_test.go

```diff
@@ -1041,13 +1040,13 @@ func TestChallenge(t *testing.T) {
 	}{
 		{
 			Name:           "Valid challenge",
-			Request:        post("valid/23"),
+			Request:        post("pending/23"),
```

Comment:
Why was this test case changed rather than added? What happens now if you post to an alread-valid challenge?